### PR TITLE
Ambr 837 mobile readability optimizations

### PR DESCRIPTION
**Pull request by samV-plos**
*JIRA issue:* https://jira.plos.org/jira/browse/AMBR-837

*Related PRs:* 
https://github.com/PLOS/plos-themes/pull/955

## What this PR does:

1. fixes the baseline typography,
2. optimizes buttons for touch, 
3. optimizes forms for touch,
4. ensures color contrast is sufficient for buttons (see screenshot),
5. deletes unused CSS
6. optimizes footer layout for touch

## Developer Notes
I tried to restore some sanity in these stylesheets, by deleting every single font-family, font-size, and line-height declaration.  (There were hundreds of declarations both in pixels and relative sizings).  

I reset the typography for `body, h1, h2, h3, h4, h5, p, small` with just a few lines of CSS. 
Please note that the fonts will seem large.  That was the intention: rather than forcing users to try and read tiny text, they can now read comfortably and scroll.  I recommend for QA to load this on an actual mobile device, instead of a simulator.

Everything should be much more usable on mobile devices. Form fields, buttons, navigational links have a minimum height and width of 44px. Specific forms that were optimized:
- Feedback
- Email this article
- Search
- Subject Filter

Other page elements:
- sidebar menu bottom section

# Code Author Tasks:

> This list should be finished before code review:
- [x] Ticket AC is updated with any decisions made in comments or side conversations.
- [x] Testing or PO Accept notes that will assist the tester, Product Owner or reviewer are set in the ticket.
- [x] All PRs are created and linked for work I have done against other projects as part of this ticket.
- [ ] Intergration Tests have been addressed.  QA has been notified of the feature and a ticket has been created for that work, or integration tests have been updated. 
- [ ] There are unit tests sufficient for both positive and negative test cases, test coverage has not decreased as part of this work. 
- [ ] Any necessary documentation in confluence or READMEs is updated.

If I used outside libraries:
- [ ] Confirmed unique - we aren't using something similar elsewhere
- [ ] License comports with our licensing (MIT preferable)

If I modified any environment variables:
- [ ] Pull requests are created to change the files in molten or in SALT-formulas.

If I made any database changes:
- [ ] DB migration scripts are created and this detail is added to testing notes in the ticket.

# Code Reviewer Tasks
- [x] I read through the JIRA ticket's AC before doing the rest of the review.
- [x] I ran the code (in the review environment or locally). I agree the running code fulfills the Acceptance Criteria as stated on the JIRA ticket. If not practical I have seen some evidence that the code does what it says (tests have passed).

### CODE REVIEW

**Comment by samV-plos** [comment]
@sgatchell Ok, develop has been merged into this branch!

File: src/main/webapp/WEB-INF/themes/mobile/ftl/common/footContent.ftl [hunk 1]
```diff
@@ -26,6 +26,6 @@
 
 <#include "siteMenu/siteMenu.ftl" />
 <#include "bodyJs.ftl" />
-
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
```
Comment (sgatchell) [inline src/main/webapp/WEB-INF/themes/mobile/ftl/common/footContent.ftl]: Is this moved down to prevent the asset from blocking the loading of JS?

Comment (samV-plos) [reply src/main/webapp/WEB-INF/themes/mobile/ftl/common/footContent.ftl]: @sgatchell correct.  I ran some audits & it appeared to be a big offender, so I moved it.

File: src/main/webapp/WEB-INF/themes/mobile/resource/css/feedback.css [hunk 2]
```diff
@@ -1,129 +1,83 @@
...
+#feedback-form-container,
+#set-grid {
+  text-align: left;
+  margin: 2rem 2%;
 }
 
-#feedback-form-container form#feedbackCreate ol, #emailThisArticleForm ol {
-  margin-top: 12px;
+#feedbackCreate ol,
+#emailThisArticleForm ol {
```
Comment (sgatchell) [inline src/main/webapp/WEB-INF/themes/mobile/resource/css/feedback.css]: why camelCase?

Comment (samV-plos) [reply src/main/webapp/WEB-INF/themes/mobile/resource/css/feedback.css]: @sgatchell those were already in place.  I suspect that they are hooked up to javascript, so I didn't want to change the id names themselves.  I only removed superfluous elements and combined rules, where possible, to reduce bloat.

File: src/main/webapp/WEB-INF/themes/mobile/resource/css/interface.css [hunk 3]
```diff
@@ -1160,27 +860,26 @@ nav.footer-secondary-menu li a {
...
 
-.browse-list a.browse-right.inactive {
+.browse-list .browse-right.inactive {
   background: #d3d3d3;
   cursor: default;
 }
 
 /* Accordions */
-.main-accordion li.accordion-item {
+.main-accordion .accordion-item {
```
Comment (sgatchell) [inline src/main/webapp/WEB-INF/themes/mobile/resource/css/interface.css]: nice, these element constraints seemed needless to me.

File: src/main/webapp/WEB-INF/themes/mobile/resource/css/interface.css [hunk 4]
```diff
@@ -70,12 +50,12 @@
   zoom: 1;
 }
 
-.bold {
-  font-weight: bold;
+.600 {
```
Comment (sgatchell) [inline src/main/webapp/WEB-INF/themes/mobile/resource/css/interface.css]: I guess it's legit in HTML5, but it sure looks odd.

Comment (samV-plos) [reply src/main/webapp/WEB-INF/themes/mobile/resource/css/interface.css]: Actually, that's unintentional! Search & replace side-effect.  I'll fix this.

File: src/main/webapp/WEB-INF/themes/mobile/resource/css/interface.css [hunk 5]
```diff
@@ -906,140 +663,98 @@ nav.footer-secondary-menu li a {
...
-  padding: 0 10px;
-  line-height: 20px;
+  height: 44px;
+  padding: 0 15px;
+  line-height: 44px;
+  text-align: center;
 }
 
-.article-item .article-options-menu > a:first-child {
+.article-option:first-child {
```
Comment (sgatchell) [inline src/main/webapp/WEB-INF/themes/mobile/resource/css/interface.css]: :+1:

**Pull request merged**